framework/doc/content/source/transfers/MultiAppGeneralFieldNearestNodeTransfer.md

```diff
@@ -0,0 +1,33 @@
+# MultiAppGeneralFieldNearestNodeTransfer
+
+!syntax description /Transfers/MultiAppGeneralFieldNearestNodeTransfer
+
+The nearest neighbor may be the nearest node or the nearest quadrature point, depending on the context
```

Comment:
I'd say the vertex average is actually about the safest place we could think of to use as a source if all we know about the source variable type is that it's not `LAGRANGE`.  That'll give a well-defined value on every FE type we have except `SIDE_HIERARCHIC`; it might even give a superconvergent value in some contexts.

Reply:
fixed